packages/components/src/Modal/Modal.tsx

```diff
@@ -39,7 +22,7 @@ export function Modal({
   secondaryAction,
   tertiaryAction,
   onRequestClose,
-}: ModalProps) {
+}: ModalLegacyProps) {
```

Comment:
The other change from the original PR was I removed the `AtlantisPortalContent` stuff for allowing the reverse theme in the Modal since other issues cropped up